tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -235,6 +235,29 @@ TEST( OutputReportTabularTest, GetUnitConversion )
 
 }
 
+TEST( OutputReportTabularTest, GetColumnUsingTabs )
+{
+	ShowMessage( "Begin Test: OutputReportTabularTest, GetColumnUsingTabs" );
+
+	std::string inString1 = " Col1 \t Col2 \t Col3 ";
+	EXPECT_EQ( " Col1 ", GetColumnUsingTabs( inString1, 1 ) );
+	EXPECT_EQ( " Col2 ", GetColumnUsingTabs( inString1, 2 ) );
+	EXPECT_EQ( " Col3 ", GetColumnUsingTabs( inString1, 3 ) );
+	EXPECT_EQ( "", GetColumnUsingTabs( inString1, 4 ) );
+
+	std::string inString2 = "Col1\tCol2\tCol3";
+	EXPECT_EQ( "Col1", GetColumnUsingTabs( inString2, 1 ) );
+	EXPECT_EQ( "Col2", GetColumnUsingTabs( inString2, 2 ) );
+	EXPECT_EQ( "Col3", GetColumnUsingTabs( inString2, 3 ) );
+	EXPECT_EQ( "", GetColumnUsingTabs( inString2, 4 ) );
+
+	std::string inString3 = "Col1\tCol2\tCol3\t";
+	EXPECT_EQ( "Col1", GetColumnUsingTabs( inString3, 1 ) );
+	EXPECT_EQ( "Col2", GetColumnUsingTabs( inString3, 2 ) );
+	EXPECT_EQ( "Col3", GetColumnUsingTabs( inString3, 3 ) );
+	EXPECT_EQ( "", GetColumnUsingTabs( inString3, 4 )  );
```

Comment:
Well, I think just being clear about what the function is expected to do and not is good.  You don't have to add support for all those, you could even put in purposely failing tests if that is easier.  If it's clear what this is supposed to cover, just make sure the function description captures that and we'll move on.